build-system/pr-check.js

```diff
@@ -164,16 +190,21 @@ function determineBuildTargets(filePaths) {
 
 const command = {
   testBuildSystem: function() {
+    const startTime = startTimer('testBuildSystem');
     execOrDie('npm run ava');
+    stopTimer('testBuildSystem', startTime);
   },
   buildRuntime: function() {
+    const startTime = startTimer('buildRuntime');
     execOrDie(`${gulp} clean`);
```

Comment:
maybe we can be even more granular by doing the timings on the execOrDie function?